test/built-ins/Symbol/species/Symbol.species.builtin-getter-name.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+ ES6 spec 'get [Symbol.species]'
+es6id: 21.2.4.2, 22.1.2.5, 22.2.2.4, 23.1.2.2, 23.2.2.2
```

Comment:
I'm not sure, but AFAIK this should be a single value.

Maybe `6.1.5.1` is the best option here to comprehend them all. Otherwise, I would split it to individual tests for each object.